blogposts/announcing-sourcegraph-3.5.md

```diff
@@ -0,0 +1,228 @@
+---
+title: 'Sourcegraph 3.5: Improved searching, repository syncing, and Bitbucket Server ACLs'
```

Comment:
```suggestion
title: 'Sourcegraph 3.5: Improved searching, repository syncing, and Bitbucket Server repository permissions'
```

"ACLs" is a wonky term that we use internally and casually, but we should be consistent and call it "repository permissions".